go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/github/hub
```

Comment:
Should we append `/v2` here? See discussion at https://github.com/github/hub/pull/1864#discussion_r214792768

Reply:
I don't think so. I don't see how this will be a breaking change for it to necessitate a change in the import path. Do we have many projects depending on hub as a library?